### PR TITLE
fix: The image preview operations are occluded

### PR DESCRIPTION
**Pull request by JarvisArt**
[[中文版模板 / Chinese template](https://github.com/ant-design/ant-design/blob/master/.github/PULL_REQUEST_TEMPLATE/pr_cn.md)]

### 🤔 This is a ...

- [ ] New feature
- [x] Bug fix
- [ ] Site / documentation update
- [ ] Demo update
- [ ] Component style update
- [ ] TypeScript definition update
- [ ] Bundle size optimization
- [ ] Performance optimization
- [ ] Enhancement feature
- [ ] Internationalization
- [ ] Refactoring
- [ ] Code style optimization
- [ ] Test Case
- [ ] Branch merge
- [ ] Other (about what?)

### 📝 Changelog

| Language   | Changelog |
| ---------- | --------- |
| 🇺🇸 English | Fixed the image preview operation being blocked |
| 🇨🇳 Chinese | 修复图像预览操作条被遮挡 |

### ☑️ Self-Check before Merge

⚠️ Please check all items below before requesting a reviewing. ⚠️

- [x] Doc is updated/provided or not needed
- [x] Demo is updated/provided or not needed
- [x] TypeScript definition is updated/provided or not needed
- [x] Changelog is provided or not needed

### CODE REVIEW

File: components/image/style/index.less [hunk 1]
```diff
@@ -105,6 +105,11 @@
...
     }
 
+    &-operations-wrapper {
+      position: fixed;
+      z-index: @zindex-image + 1;
+    }
+
     &-operations {
       .reset-component();
       position: fixed;
```
Comment (afc163) [inline components/image/style/index.less:115]: 里面的元素是不是不需要 fixed 了？

Comment (JarvisArt) [reply components/image/style/index.less:115]: 里面的元素可以不需要fixed，需要把 &-operations  以下样式放在  &-operations-wrapper 里
 `position: fixed;`
` top: 0;`
 `right: 0;`
 `z-index: @zindex-image + 1;`
 `width: 100%;`

Comment (afc163) [reply components/image/style/index.less:115]: 那这样里面改成 position: absolute 就行了

Comment (JarvisArt) [reply components/image/style/index.less:115]: 试了下，&-operations 的 `position` 不为 `fixed `，&-operations-wrapper 还是一样要加上面那五个样式。

Comment (afc163) [reply components/image/style/index.less:115]: 改呗，统一放在外层比较合理。

Comment (JarvisArt) [reply components/image/style/index.less:115]: 👌🏻

Comment (JarvisArt) [reply components/image/style/index.less]: 发现 相册模式 例子有点问题，operations显示不出来了

File: components/image/style/index.less [hunk 2]
```diff
@@ -105,16 +105,19 @@
...
+    &-operations-wrapper {
       position: fixed;
       top: 0;
       right: 0;
       z-index: @zindex-image + 1;
+      width: 100%;
+    }
+
+    &-operations {
+      .reset-component();
```
Comment (afc163) [inline components/image/style/index.less:117]: switch-left 和 switch-right 也应该改成 `postiion: absolute;`。

Comment (afc163) [reply components/image/style/index.less:117]: wrapper `height: 100%`

Comment (JarvisArt) [reply components/image/style/index.less:117]: wrapper `height: 100%` 会挡住图片的，图片没法移动

**Review by afc163 — APPROVED** [review]

**Pull request merged**